### ISSUE TITLE
Suggestions for `defmulti`, `defmethod` and `defprotocol`

### ISSUE THREAD

**Issue by tomasz-biernacki** [Lambda-X/re-complete#13]
There are none.
![hn5zeniqd6](https://cloud.githubusercontent.com/assets/15321488/14880802/2ca57da8-0d31-11e6-8be2-5c7f6b624c8c.gif)

**Comment by herichovadajana** [comment]
Yes that's because `defmulti` `defmethod` and `defprotocol` are not in `cljs.core` ns
There are only all funcions from `cljs.core`. But maybe we can add there some another namespaces from `clojurescript`. 

@tomasz-biernacki @arichiardi
